simulation_parameters/SimulationParameters.cs

```diff
@@ -222,34 +222,48 @@ public GameCredits GetCredits()
         return gameCredits;
     }
 
-    public OrganelleDefinition GetRandomProkaryoticOrganelle(Random random)
+    public OrganelleDefinition GetRandomProkaryoticOrganelle(Random random, bool lawkOnly)
     {
         float valueLeft = random.Next(0.0f, prokaryoticOrganellesTotalChance);
 
-        foreach (var organelle in prokaryoticOrganelles)
+        // Filter to only LAWK organelles if necessary
+        IEnumerable<OrganelleDefinition> usedOrganelles = prokaryoticOrganelles;
+        if (lawkOnly)
+            usedOrganelles = usedOrganelles.Where(o => o.Lawk);
+
+        OrganelleDefinition chosenOrganelle = null!;
```

Comment:
Don't do nullability suppression like this. It is only meant for constructor-like resources (like Godot _Ready). In this case ~~I think you can just leave this variable uninitialized, or~~ just assign null (you need to make the variable nullable first `OrganelleDefinition?`). And then after the loop if the variable is null throw an exception as it's invalid that there wasn't any organelles.